packages/core/test/menu/menuTests.tsx

```diff
@@ -151,39 +151,42 @@ describe("MenuItem", () => {
         afterEach(() => ReactDOM.unmountComponentAtNode(childContainer));
 
         it("children can display left", done => {
-            menuItem = ReactDOM.render(
-                <MenuItem iconName="style" text="Style">
+            menuItem = mountMenuItem(
+                { iconName: "style", text: "Style" },
+                // use a fragment instead of an array, so we don't have to key each item
+                <React.Fragment>
                     <MenuItem iconName="bold" text="Bold" />
                     <MenuItem iconName="italic" text="Italic" />
                     <MenuItem iconName="underline" text="Underline" />
-                </MenuItem>,
-                childContainer,
-            ) as MenuItem;
+                </React.Fragment>,
+            );
             hoverOverTarget(0, () => {
                 assert.isNotNull(childContainer.query(`.${Classes.ALIGN_LEFT}`));
                 done();
             });
         });
 
         it("useSmartPositioning=false prevents display left behavior", done => {
-            menuItem = ReactDOM.render(
-                <MenuItem iconName="style" text="Style" useSmartPositioning={false}>
+            menuItem = mountMenuItem(
+                { iconName: "style", text: "Style", useSmartPositioning: false },
+                <React.Fragment>
                     <MenuItem iconName="bold" text="Bold" />
                     <MenuItem iconName="italic" text="Italic" />
                     <MenuItem iconName="underline" text="Underline" />
-                </MenuItem>,
-                childContainer,
-            ) as MenuItem;
+                </React.Fragment>,
+            );
             hoverOverTarget(0, () => {
                 assert.isNotNull(childContainer.query(`.${Classes.OVERLAY_OPEN}`));
                 assert.isNull(childContainer.query(`.${Classes.ALIGN_LEFT}`));
                 done();
             });
         });
 
-        it("children will continue displaying in the same direction if possible", done => {
-            menuItem = ReactDOM.render(
-                <MenuItem iconName="style" text="Style">
+        // TODO (clewis): Commented this out during the Popover2 => Popover refactor. Get this working again.
```

Comment:
Never got this test working, even though the functionality works. Should I try again now?

Reply:
yes, but it shouldn't block this PR. please file a follow-up ticket and add it to the 2.0.0 milestone

Reply:
https://github.com/palantir/blueprint/issues/2008